rust-sdk/whirlpool/src/account.rs

```diff
@@ -0,0 +1,67 @@
+use solana_account_decoder::UiAccountEncoding;
+use solana_client::{
+    rpc_client::RpcClient,
+    rpc_config::{RpcAccountInfoConfig, RpcProgramAccountsConfig},
+    rpc_filter::{Memcmp, RpcFilterType},
+};
+use solana_program::pubkey::Pubkey;
+use solana_sdk::{program_pack::Pack, rent::Rent};
+use spl_token_2022::state::Account;
+use std::error::Error;
+
+#[derive(Debug, Clone)]
+pub struct RpcKeyedTokenAccount {
+    pub pubkey: Pubkey,
+    pub token_program: Pubkey,
+    pub token: Account,
+}
+
+// This is a little hacky but it is done this way because
+// the original get_token_accounts_for_owner uses json_parsed encoding
+// and we don't want to use serde_json in this crate.
+pub(crate) fn get_token_accounts_for_owner(
```

Comment:
Got it. Might be safer to just switch it back to the actual rpc call in that case

Reply:
Yep, I think it is safer.
I tried Helius RPC and clearly rejected immediately.
```
SolanaJSONRPCError: failed to get accounts owned by program TokenkegQfeZyiNwAJbNbGKPFXCWuBvf9Ss623VQ5DA: TokenkegQfeZyiNwAJbNbGKPFXCWuBvf9Ss623VQ5DA excluded from account secondary indexes; this RPC method unavailable for key
```
I also tried Shyft RPC. It didn't reject it, but request was timed out 😅 (It makes sense because there are too much accounts)